nin/backend/render.js

```diff
@@ -1,21 +1,42 @@
-var spawn = require('child_process').spawn;
+var p = require('path');
+var ffmpeg = require('fluent-ffmpeg');
 
 var render = function(projectPath) {
-  var avconv = spawn('avconv', [
-  '-y',
-  '-r', '60',
-  '-i', projectPath + '/bin/render/%07d.png',
-  '-i', projectPath + '/res/music.mp3',
-  '-c:v', 'libx264',
-  '-c:a', 'copy',
-  '-crf', '16',  
-  projectPath + '/bin/render/render.mp4']);
-  avconv.stdout.on('data', function(data) {
-    console.log(data.toString());
-  });
-  avconv.stderr.on('data', function(data) {
-    console.log(data.toString());
-  });
-}
+  console.time('Total execution time');
+
+  var pngPath = p.join(projectPath, '/bin/render/%07d.png');
+  var musicPath = p.join(projectPath, '/res/music.mp3');
+  var destination = p.join(projectPath, '/bin/render/demo.mp4');
+
+  ffmpeg()
+    .addInput(pngPath)
+    .withInputFps(60)
+
+    .addInput(musicPath)
+    .audioCodec('copy')
+
+    .videoCodec('libx264')
+    .withOutputFps(60)
+    .withOutputOption('-pix_fmt', 'yuv420p') // YUV color space
```

Comment:
No, `-y` means overwrite if the file already exists. It seems that I don't need to specify this explicitly, because it doesn't complain about any existing file.

`pix_fmt`, however, specifies the pixel format. yuv420p is made with human perception in mind, so that's good.